src/utils/API.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+const gamersAPI = axios.create({
+  baseURL: "https://bakrisncgames.onrender.com/api",
+});
+
+export const getReviews = () => {
+  return gamersAPI.get("/reviews").then((res) => {
+    return res.data;
+  });
+};
+
+export const patchVotesInc = (reviewId) => {
+  const incrementVote = { inc_votes: 1 };
+  return gamersAPI
+    .patch(`/reviews/${reviewId}`, incrementVote)
+    .then(({ data }) => {});
+};
```

Comment:
These functions aren't DRY, you could condense these into a single function. As suggested with the voting components in the comments above. There's no need to have the promise resolve to undefined. If you don't want to make use of the response data, you can remove the .then 👍